### PR TITLE
New Species: Harpy

### PR DESCRIPTION
**Pull request by kcin2001**
Adds harpies the only thing special about them is they have nevrean legs on humans bodies and starts with harpy wings and a bird tail
also adds markings to color the feet and thighfeathers (separately) 
i have tested to make sure they function as intended

### CODE REVIEW

**Comment by SpadesNeil** [comment]
I don't want them to be called "Harpies" in-game. Let's give them a proper naming.

"Rapala" is what you used. "Rapalas" was the plural form? You sure you did not mean "Rapalans"?

**Comment by kcin2001** [comment]
i did not write the lore blurb but i will change the name to rapalas and rapalans

**Comment by kcin2001** [comment]
What i think i have to change some stuff now that the name has changed

**Comment by kcin2001** [comment]
Ill try to remember to do it in the morn

**Comment by SpadesNeil** [comment]
also needs a lore page

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm [hunk 1]
```diff
@@ -331,3 +331,40 @@
...
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/claws, /datum/unarmed_attack/bite/sharp)
+	num_alternate_languages = 3
+	secondary_langs = list(LANGUAGE_BIRDSONG)
+	name_language = null
+	color_mult = 1
+
+	min_age = 18
+	max_age = 80
+
+	blurb = "An Avian species, coming from a distant planet, the Rapalas are the very proud race.\
```
Comment (Yoshax) [inline code/modules/mob/living/carbon/human/species/station/station_vr.dm]: There's some pretty big mistakes in this blurb. It needs thought out and written properly.

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm [hunk 2]
```diff
@@ -331,3 +331,40 @@
...
+	spawn_flags = SPECIES_CAN_JOIN
+	appearance_flags = HAS_HAIR_COLOR | HAS_LIPS | HAS_UNDERWEAR | HAS_SKIN_COLOR | HAS_EYE_COLOR
+
+	//flesh_color = "#AFA59E"
+	//base_color = "#777777"
+	//tail_hair = "feathers"
+	//reagent_tag = IS_SERGAL
+
+	heat_discomfort_strings = list(
+		"Your feathers prickles in the heat.",
```
Comment (Yoshax) [inline code/modules/mob/living/carbon/human/species/station/station_vr.dm]: "Your feathers prickle in the heat."*

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm (hunk 1 above)
Comment (kcin2001) [reply code/modules/mob/living/carbon/human/species/station/station_vr.dm]: What mistakes are you talking about, though that does remind me i should remove some of the attacks

**Comment by kcin2001** [comment]
i added my species to the species page on the wiki ace, i did notice that vassalians need a portion on the page

File: code/modules/mob/living/carbon/human/species/station/station_vr.dm (hunk 1 above)
Comment (Yoshax) [reply code/modules/mob/living/carbon/human/species/station/station_vr.dm]: Something more like:
```blurb = "An Avian species, coming from a distant planet, the Rapalans are a very proud race.\
Sol researchers have commented on them having a very close resemblance to the mythical race called 'Harpies',\
who are known for having massive winged arms and talons as feet. They've been clocked at speeds of over 35 miler per hour chasing the planet's many fish-like fauna.\
The Rapalan's home-world 'Verita' is a strangely habitable gas giant, while no physical earth exists, there are fertile floating islands orbiting around the planet from past asteroid activity."```

Would be better. There were quite a few grammar mistakes and such in there.

Comment (kcin2001) [reply code/modules/mob/living/carbon/human/species/station/station_vr.dm]: Thank you im very poor at catching those kinds of things in other people's work and i thought i had got them all with a revision someone else provided

**Comment by kcin2001** [comment]
is there any more issues with my pr, anything i should look at or know, should i post a pic of the harpy?

**Comment by kcin2001** [comment]
It works and should be fully ready

**Comment by SpadesNeil** [comment]
Where's the link to the lore page?

**Comment by kcin2001** [comment]
I put it on the species page and it is the same blurb thats in game

**Comment by kcin2001** [comment]
I did fix the mistakes yosh pointed out i thought

**Comment by SpadesNeil** [comment]
Oh, well where's the wiki page?

**Comment by kcin2001** [comment]
https://wiki.vore-station.net/Species

**Comment by kcin2001** [comment]
i dont really have any extensive species lore so it does not have its own page

**Comment by kcin2001** [comment]
can you please tell me why this is still marked dnm

**Comment by SpadesNeil** [comment]
Wrong wiki page. You want: https://wiki.vore-station.net/Backstory

Also there's no images of the new species.

**Comment by kcin2001** [comment]
Cameron told me about the backstory page since you didnt and ill add a pic to my pr now since you didnt tell me last time

**Comment by kcin2001** [comment]
https://wiki.vore-station.net/Backstory#Rapala 

image
image
image
@SpadesNeil

**Comment by SpadesNeil** [comment]
Later on I want to see that lore page fleshed out.

**Comment by kcin2001** [comment]
I am piss poor at writing lore, i didnt even write the lore blurb myself someone in the discord helped me

**Pull request merged**